src/main/java/mekanism/common/CommonPlayerTickHandler.java

```diff
@@ -179,21 +181,22 @@ public static boolean handleJetpackMotion(Player player, JetpackMode mode, Boole
         return true;
     }
 
-    private static boolean isJetpackOn(Player player, ItemStack chest) {
+    private static ItemStack getJetpackIfOn(Player player) {
+        final ItemStack chest = ItemJetpack.getJetpack(player);
```

Comment:
Why the change to this? The idea of it taking the chest stack as a parameter was to not have to relook up the stored item.